eng/Versions.props

```diff
@@ -230,6 +230,7 @@
     <!-- Packages from 2.1/2.2 branches used for site extension build -->
     <MicrosoftAspNetCoreAzureAppServicesSiteExtension21PackageVersion>2.1.1</MicrosoftAspNetCoreAzureAppServicesSiteExtension21PackageVersion>
     <MicrosoftAspNetCoreAzureAppServicesSiteExtension22PackageVersion>2.2.0</MicrosoftAspNetCoreAzureAppServicesSiteExtension22PackageVersion>
+    <MicrosoftAspNetCoreAzureAppServicesSiteExtension31PackageVersion>3.1.3</MicrosoftAspNetCoreAzureAppServicesSiteExtension31PackageVersion>
```

Comment:
Unfortunately we can't find this reference until the next 3.1 patch is released.

Reply:
Can't we use 3.1.2 in the meantime?

Reply:
No, because we weren't producing a package until a change we made a few days ago

Reply:
Shoot